src/validate.rs

```diff
@@ -774,6 +776,64 @@ fn validate_repos(data: &Data, errors: &mut Vec<String>) {
     });
 }
 
+/// Enforce that website roles are only assigned to a valid team member, and
+/// that the same role id always has a consistent description across teams
+/// (because the role id becomes the Fluent id used for translation).
+fn validate_website_roles(data: &Data, errors: &mut Vec<String>) {
+    let mut role_descriptions = HashMap::new();
+
+    wrapper(
+        data.teams().chain(data.archived_teams()),
+        errors,
+        |team, errors| {
+            let team_name = team.name();
+            if let Some(website_data) = team.website_data() {
```

Comment:
Nit: might help if we try to reduce the rightward drift a bit:

```suggestion
           let Some(website_data) = team.website_data() else { return Ok(()); };
```

Reply:
Done. Good call.